BEIMA.Client/services/ApiClient.js

```diff
@@ -0,0 +1,24 @@
+import axios from 'axios';
```

Comment:
It looks like you have the same code in three separate files, is this intentional?

Reply:
It might not be picking up that the original "apiclient" file was renamed to "GetDevice", and then "GetDevice" was moved from /services to /src/services.  The remove of apiclient and move of getdevice might not have been added to the commit. I'll have to check once I'm off work.

Reply:
Just deleted them through the web interface, one file, where it's supposed to be, is included